packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.spec.tsx

```diff
@@ -1271,4 +1309,191 @@ describe("viewport positioner", (): void => {
         expect(positioner.instance().baseHorizontalOffset).toBe(-10);
         expect(positioner.instance().baseVerticalOffset).toBe(-10);
     });
+
+    test("Positioner scaled sizes calculated correctly", (): void => {
```

Comment:
I think it would be preferable to say *what* was calculated correctly.
Example "should scale the height to match the viewports height when scaleToAvailableSpace is set to true"

Reply:
yep